app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -43,12 +43,24 @@ def save_proofing_components
         Db::ProofingComponent::Add.call(user_id, :liveness_check, DocAuthRouter.doc_auth_vendor)
       end
 
+      # @param [IdentityDocAuth::Response,
+      #   DocumentCaptureSessionAsyncResult,
+      #   DocumentCaptureSessionResult] response
       def extract_pii_from_doc(response)
         current_user = User.find(user_id)
         flow_session[:pii_from_doc] = response.pii_from_doc.merge(
           uuid: current_user.uuid,
           phone: current_user.phone_configurations.take&.phone,
         )
+        if response.respond_to?(:extra)
+          # Sync flow: IdentityDocAuth::Response
+          flow_session[:document_expired] = response.extra&.dig(:document_expired)
+          flow_session[:reproof_at] = response.extra&.dig(:reproof_at)
```

Comment:
Do we need both values in the session? Would it be enough to retain only `document_expired`, and use that as a trigger in `IdvSession` to set `reproof_at` as the value of `IdentityConfig.store.proofing_expired_license_reproof_at.to_s`?

Reply:
That's a good point, I guess we don't. I'll make a pass to clean this up